walkthroughs/week-2-server/examples/text-processor/src/main/java/com/google/sps/servlets/TextProcessorServlet.java

```diff
@@ -28,9 +28,9 @@
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get the input from the form.
-    String text = request.getParameter("text-input");
-    boolean upperCase = Boolean.valueOf(request.getParameter("upper-case"));
-    boolean sort = Boolean.valueOf(request.getParameter("sort"));
+    String text = getParameter(request, "text-input", "");
+    boolean upperCase = Boolean.parseBoolean(getParameter(request, "upper-case", "false"));
+    boolean sort = Boolean.parseBoolean(getParameter(request, "sort", "false"));
```

Comment:
According to Java naming conventions (#14 on https://www.cs.bgu.ac.il/~majeek/presentations/JavaProgrammingStyle%20Guidelines.html), booleans should be named as "isUpperCase", "isSorted". 

As a side note, this follows in a lot of other languages and makes more clear when you use that variable later in the code that it is a boolean.

Reply:
I didn't write this code, this code came with the downloaded files